### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by hheavenknowss** [milesial/Pytorch-UNet#255]
Hi there, thx for your work! it's very helpful.
But I have an issue about loss function. In my problem, I have a liver cancer  segmentation mission, I set anything right and I don't do any change about code, and I find that the value of loss is usually about 0.0xx.
After 5 epochs training, I do a predict, and the results is all black.
here is my vision of loss.
![image](https://user-images.githubusercontent.com/35052711/104730140-7541c700-5774-11eb-92ec-7f5804955846.png)
do you know what's happend?

**Comment by milesial** [comment]
Your learning rate seems too high. Spend more time on hyperparameter tuning, and if possible, use a larger batch size.
